README.md

```diff
@@ -103,9 +102,8 @@ cast call 0xe7f1725e7734ce288f8367e1bb143e90bb3f0512 'fibonacci(uint256)' 5
 
 ## Proving Modes
 The foundry template supports three different proving modes:
-1. `none` - By default, only the [executor](https://www.dev.risczero.com/docs/terminology#executor) runs your zkVM program and no proof is generated. Because there is no proving, this will be the fastest way to test. 
-2. `local` - A proof of execution generated on your local machine. 
-3. `bonsai` - A proof of execution is generated by the Bonsai API.
+1. `local` - By default, only the [executor](https://www.dev.risczero.com/docs/terminology#executor) runs your zkVM program and no proof is generated. Because there is no proving, this will be the fastest way to test. 
+2. `bonsai` - A proof of execution is generated by the Bonsai API.
 
 https://github.com/risc0/bonsai-foundry-template/assets/15272444/db2b25c0-61b1-4884-9f8e-1d3641decf4a
```

Comment:
We should remove this video for now since we've diverged sufficiently far that it will be confusing.
```suggestion
```